vagrant/tutorial/bootstrap.py

```diff
@@ -29,11 +29,6 @@ def main():
     # Install a repository that provides epel packages/updates.
     yum_install("epel-release")
 
-    # The kernel was just upgraded which means the existing VirtualBox Guest
-    # Additions will no longer work.  Build them again against the new version
-    # of the kernel.
-    check_output(["/etc/init.d/vboxadd", "setup"])
-
     # Create the 'docker' group.  The Vagrant feature for pulling Docker images
```

Comment:
this is handled by the vagrant VBguest plugin, and the chunk of code in the Vagrantfile